src/makeCalculator.js

```diff
@@ -33,11 +33,63 @@
  *
  * calculator.result === -28
  *
- *
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const add = num => {
+    result += num;
+
+    return this;
+  };
+
+  const subtract = num => {
+    result -= num;
+
+    return this;
+  };
+
+  const multiply = num => {
+    result *= num;
+
+    return this;
+  };
+
+  const divide = num => {
+    if (num === 0) {
```

Comment:
Dividing by zero is not handled correctly. It's better to throw an error instead of just returning 'this'.